src/pages/Main/Components/LectureSlide.js

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import { useEffect, useState } from 'react';
+import styled from 'styled-components';
+import Slider from 'react-slick';
+import Card from '../../../components/Card/Card';
+
+const LectureSlide = () => {
+  const [cardList, setCardList] = useState([]);
+  const getData = async () => {
+    const data = await (await fetch('/data/cardList.json')).json();
+
```

Comment:
```suggestion
    const response = await fetch('/data/cardList.json');
    const data = await response.json();
```
() 이용해서 한줄로 처리하는 방법도 좋겠지만 위와 같이 나누어서 처리할 수도 있겠네요, 저는 개인적으로 가독성 측면에서 나눠서 적는것이 좋아보인다고 생각하는데 이부분은 개인마다, 회사마다, 팀마다 의견 및 컨벤션이 다를 수도 있어서 참고만 해주세요!

Reply:
네 ! 우선 수정하였습니다!